bridge.gemspec

```diff
@@ -0,0 +1,34 @@
+# coding: utf-8
+lib = File.expand_path('../lib', __FILE__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require 'bridge/version'
+
+Gem::Specification.new do |spec|
+  spec.name          = "bridge"
```

Comment:
Yep. Conventionally matching the root namespace. You can change this, however. 

And we will want to, if we want to avoid colliding with other people who had the same idea: https://rubygems.org/gems/bridge